File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;
class Solution {
    public Integer[] solution(int[] numbers) {

        ArrayList<Integer> list = new ArrayList<>();
        for(int i = 0; i< numbers.length; i++){
            for(int j = i+1; j< numbers.length; j++){
                int sum = numbers[i]+numbers[j];
                if(!(list.contains(sum))){
                    list.add(sum);
                }
            }
        }
        Collections.sort(list);
        Integer[] answer = list.toArray(new Integer[0]);
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
#include <set>


using namespace std;

vector<int> solution(vector<int> numbers) {
    set<int> sumSet;
    for(int i = 0; i< numbers.size(); i++){
        for(int j = i+1; j< numbers.size(); j++){
            int sum =  numbers[i]+numbers[j];
            sumSet.insert(sum);
        }
    }
    vector<int> answer (sumSet.begin(),sumSet.end());
    return answer;
}